src/pages/Script/styled.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
```

Comment:
The use of styled components is not recommended for this project, and module css is preferred for the remainder。